src/Queue/RabbitMQQueue.php

```diff
@@ -131,10 +138,26 @@ public function pop($queueName = null)
                 return new RabbitMQJob($this->container, $this, $consumer, $message);
             }
         } catch (\Exception $exception) {
+            if ($exception instanceof AMQPConnectionClosedException ||
+                $exception instanceof AMQPChannelClosedException
```

Comment:
This can be in a separate catch block:

```
catch(AMQPConnectionClosedException | AMQPChannelClosedException $exception) {
    // ...
}
```

Reply:
Fixed and I added a PR for the master branch also.